source/gloperate-assimp/CMakeLists.txt

```diff
@@ -0,0 +1,129 @@
+
+# Target
+set(target gloperate-assimp)
+message(STATUS "Lib ${target}")
+
+
+#
+# External libraries
+#
+
+find_package(GLM REQUIRED)
+find_package(glbinding REQUIRED)
+find_package(globjects REQUIRED)
+find_package(libzeug REQUIRED)
+find_package(ASSIMP)
+
+# Build only if ASSIMP is available
+if (NOT ASSIMP_FOUND)

```

Comment:
Our current default is to omit the "Lib ${target}" output if the target won't be created (see https://github.com/hpicgs/glbinding/blob/master/source/examples/cubescape-qt/CMakeLists.txt)